app/models/ems_refresh/save_inventory_cinder.rb

```diff
@@ -4,15 +4,7 @@
 #   - cloud_volumes
 #   - cloud_volume_backups
 #   - cloud_volume_snapshots
-#   - vms
-#     - storages (link)
-#     - security_groups (link)
-#     - operating_system
-#     - hardware
-#       - disks
-#       - guest_devices
-#     - custom_attributes
-#     - snapshots
+#   - backing_links
 #
 
 module EmsRefresh::SaveInventoryCinder
```

Comment:
I don't understand this comment at all.  The save_inventory code is a generic saving structure that maps to the database.  Since you are going into the same tables, it is automatically.

Now the *parser* will of course be different, and may or may not populate different parts of the hash structure, but the save_inventory code should be the same.